### PR TITLE
Startup scripts

### PR DESCRIPTION
**Pull request by TheDarkPyotr**
- Added observability stack _(Grafana, Loki, Promtail)_ to `root-orchestrator.yml` & `cluster-orchestrator.yml`
- Rename to `dashboard.json` the root and cluster dashboard to avoid refactoring `downloadConfigFiles.sh` under `/scripts` folder
- Fix minor typos in Prometheus downloading conf file

### CODE REVIEW

File: scripts/utils/downloadConfigFiles.sh
Comment (giobart) [inline scripts/utils/downloadConfigFiles.sh:1]: why did you remove the done in the for loop?

Comment (TheDarkPyotr) [reply scripts/utils/downloadConfigFiles.sh:1]: My bad! Totally unintentional change: fixed

**Review by giobart — APPROVED** [review]

**Comment by giobart** [comment]
@TheDarkPyotr LGTM, can we merge it?
